osg-run-tests

```diff
@@ -49,9 +51,27 @@ set_parameters_base()
     esac
 }
 
+GH_ACCOUNT=opensciencegrid
+GH_BRANCH=master
+
+set_github_info()
+{
+    if [[ "$1" =~ [A-Za-z0-9_-]*:[A-Za-z0-9_-]* ]]; then
```

Comment:
I think you mean

`[[ $1 =~ ^[A-Za-z0-9_-]*:[A-Za-z0-9_-]*$ ]]`

For the `[[ str =~ regex ]]` matches, the regex does not have to match the whole string.

(As it stands, that check will match anything containing a `:`)